### ISSUE TITLE
Evidence negative control generation hangs (v2.3)

### ISSUE THREAD

**Issue by anthonysena** [OHDSI/Atlas#537]
### Expected behavior

Specifying a large list of concepts to include/exclude as part of the evidence processing and the output should indicate that these are user included/excluded concepts.

### Actual behavior

Specifying a large list of concepts to include/exclude as part of the evidence processing. Generate results  but the processing never completes.

### Steps to reproduce behavior

See above.

**Comment by ericaVoss** [comment]
Example exclude list 

```{TEXT}
194913,372888,373470,374032,374925,375260,375806,376112,376395,376686,378140,378420,380394,381551,4027396,4049810,40480668,43531027,43531028,
255841,257905,258780,261889,4050731,4112826,4137525,4138392,4166517,4172303,4230358,4236366,4309350,40481763,46270376,
252351,252548,252655,252949,253235,253790,254066,254266,254561,254576,254677,255084,255735,255848,256722,256723,257315,257908,258061,258180,258333,258354,258785,259048,259852,259992,260028,260041,260430,260754,261053,261324,261326,436145,437313,439857,440431,442637,443410,4021760,4025165,4026113,4044215,4045227,4045260,4046011,4048052,4048147,4048148,4048149,4048517,4048518,4048519,4049279,4049965,4050872,4050874,4051333,4051334,4051335,4051336,4051337,4051338,4051339,4052546,4052547,4052548,4070540,4071610,4071611,4080435,4080753,4080883,4082065,4084973,4102253,4110039,4110506,4110507,4110509,4110510,4111119,4112655,4112662,4112820,4112822,4114030,4114031,4116487,4116488,4117114,4119431,4119436,4119795,4124539,4133224,4135197,4137435,4138244,4138769,4140134,4141619,4143092,4144107,4145369,4148529,4153356,4166072,4169796,4171251,4174281,4174308,4174309,4175598,4177385,4186568,4190647,4193964,4195014,4195452,4200891,4203846,4204819,4205578,4212120,4215807,4221503,4221767,4222062,4223032,4225318,4228277,4233319,4236311,4237183,4240452,4245006,4245499,4245828,4248029,4248154,4248807,4256236,4256894,4267135,4273378,4274802,4274981,4276663,4280213,4284985,4293463,4294404,4299862,4308451,4309106,4310964,4311555,4322625,4327820,4334649,4341520,4345215,4345699,4347659,36714118,37016927,37017277,37017278,37019058,37394479,40479642,40480033,40481335,40481839,40482061,43020558,44782989,45757206,45757250,45757644,45763749,45763750,45763751,45763752,45767051,45768914,45768960,45768961,45768997,45768998,45769809,45770900,45771022,46269693,46269707,46269708,46269709,46269710,46269711,46269712,46269713,46269714,46269715,46269716,46269717,46269718,46269719,46269720,46269721,46269722,46269723,46269724,46269725,46269726,46269954,46270027,46270121,46270128,46270318,46270510,46274035
```

**Comment by anthonysena** [comment]
So this bug is happening because the list of concept_ids is > 250 characters and when attempting to use this list as a parameter to the Spring Batch job for processing negative controls, it fails because it cannot insert a parameter > 250 characters into the database. We will need to change the interface to allow for a user to specify a concept set that represents the list of concepts to include/exclude and then resolve the concept set via WebAPI.

**Comment by anthonysena** [comment]
Closed as part of the Atlas/WebAPI 2.5.0 release.
